src/base.ts

```diff
@@ -0,0 +1,29 @@
+import { Reshuffle, BaseHttpConnector } from 'reshuffle-base-connector'
+import { Client, ClientOptions } from '@microsoft/microsoft-graph-client'
+
+import { ClientCredentialAuthenticationProvider, MicrosoftOptions } from './auth'
+
+export type MicrosoftConnectorConfigOptions = MicrosoftOptions
```

Comment:
Not used outside this here and not exported in `/src/index.ts` as well?

Reply:
Will export it from `index.ts`